app/clients/zendesk_sell.py

```diff
@@ -55,21 +60,44 @@ def _generate_lead_data(contact: ContactRequest) -> Dict[str, Union[str, List[st
             }
         }
 
-    def send_contact_request(self, contact: ContactRequest) -> int:
-        ret = 200
-        if contact.is_demo_request():
-            ret = self.send_lead(contact)
+    @staticmethod
+    def _generate_contact_data(user: User) -> Dict[str, Union[str, List[str], Dict]]:
 
-        return ret
+        # validation based upon api mandatory fields
+        assert len(user.name) and len(user.email_address), 'Name or email are mandatory field'
+
+        first_name, last_name = ZenDeskSell._name_split(user.name)
+        return {
+            'data': {
+                'last_name': last_name,
+                'first_name': first_name,
+                'email': user.email_address,
+                'mobile': user.mobile_number,
```

Comment:
Does the phone number need to be transformed, maybe using the international format?

Reply:
My opinion is that we want to match the Zendesk Integration directly to the data that we have within our Notify database. I'd like to keep the data as a 1-to-1 mapping as much as possible.

If the better format is to transform it to an international number format I'd also recommend also modifying the mobile number at form inject to the international format within our database. WDYT?